be/src/vec/columns/column_dictionary.h

```diff
@@ -158,10 +158,6 @@ class ColumnDictionary final : public COWHelper<IColumn, ColumnDictionary<T>> {
         __builtin_unreachable();
     }
 
-    bool is_fixed_and_contiguous() const override { return true; }
```

Comment:
is this can be deleted ? 
because get_raw_data() maybe depends on it?!

Reply:
now , only use in array.